src/components/propositionalLogic/wiki.tsx

```diff
@@ -20,17 +20,36 @@ class WikiPropositionalLogic extends React.Component {
 				<p>Atomare Formeln können mit den Junktoren
 				&#172; (nicht), &amp; (und), | (oder), &rarr; (Implikation) und &harr; (Äquivalenz).
 				&#172; negiert den Wahrheitswert einer Formel. Das heißt eine wahre Formel wird falsch
-				und eine falsche Formel wird wahr. &#172; ist der
-				einzige Junktor, der nur auf einer statt auf zwei Formeln
-				operiert.
+				und eine falsche Formel wird wahr. Statt &amp; sieht man normalerweise
+				das Zeichen &#8743; und statt | das Zeichen &#8744;. Auf einer gewöhnlichen Tastatur
+				gibt es diese Zeichen jedoch nicht. Daher benutzen wir in Diproche lieber die ersten
+				beiden Zeichen.
+				&#172; ist der einzige Junktor, der nur auf einer statt auf zwei Formeln operiert.
 				Wenn A und B Formeln sind, dann ist A&amp;B wahr genau dann, wenn
 				sowohl A, als auch B wahr ist. Ansonsten ist A&amp;B falsch.
 				A | B ist wahr genau dann, wenn mindestens eine der Formeln
 				A und B wahr ist. Ansonsten ist A | B falsch.
-				A&arr;B ist wahr genau dann, wenn entweder die Voraussetzung A
-				falsch ist oder A und B wahr sind. Ansonsten ist A&arr;B falsch.
+				A&rarr;B ist wahr genau dann, wenn entweder die Voraussetzung A
+				falsch ist oder A und B wahr sind. Ansonsten ist A&rarr;B falsch.
 				A&harr;B ist wahr genau dann, wenn A und B beide falsch oder beide
 				wahr sind. Ansonsten ist A&harr;B falsch.</p>
+				<h2>Klassifikation einige Formeln</h2>
+				<p> Sei F eine Formel mit n Aussagenvariablen. Man kann allen Aussagenvariablen
+					von F auf 2<sup>n</sup> verschiedene Weisen einen Wahrheitswert zuordnen.
+					F heißt nun <b>allgemeingültig</b>, falls jede Wertebelegung F wahr macht.
```

Comment:
Hier gibt es noch weitere Begriffe, die man erwähnen kann. Ich denke Frau Krapf nutzt noch Tautologie und Kontradiktion in der EM Vorlesung.

Reply:
Das stimmt, werde ich noch hinzufügen.